### ISSUE TITLE
Parsing <input> "checked" and "value"

### ISSUE THREAD

**Issue by christianfredh** [remarkablemark/html-react-parser#321]
Parsing `<input type="checkbox" checked />` gives this warning:
You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.

Parsning `<input type="text" value="initial value" />` gives this warning:
You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.

We made an attempt to fix in the replace method:

```
if (node.type === 'tag' && node.name === 'input') {
	  if (node.attribs.type === 'text') {
	    node.attribs.defaultValue = node.attribs.value
	    delete node.attribs['value']
	  }
	  
	  if (node.attribs.type === 'checkbox') {
	    node.attribs.defaultChecked = node.attribs.checked
	    delete node.attribs['checked']
	  }
}
```

This seems to work, but not sure if it's the best. Maybe it would be good for support for this case in the package? I see there is something similar dom-to-react for textarea.

**Comment by remarkablemark** [comment]
Sorry for the late response @christianfredh, but does the warning show up for all `input` types and `textarea`? Also, does this warning change the expected behavior of the elements? If so, then it might make sense for the package to handle.

Do you think there would be any downside if `defaultValue` and `defaultChecked` are always set for DOM nodes with `value` and `checked` attributes?

**Comment by remarkablemark** [comment]
Bug will be fixed in #388

**Comment by remarkablemark** [comment]
Released in #389

**Comment by christianfredh** [comment]
Sorry, didn't have a chance to look into it again. It looks like a good change.
